timemory/roofline/roofline.py

```diff
@@ -349,15 +410,111 @@ def get_peak_bandwidth(roof_data, band_labels):
     for (band, band_info) in zip(band_list, band_info_list):
         band_info = band_info + " GB/s"
         peak_bandwidths.append([float(band[0]/band[1]), band_info])
+
     return peak_bandwidths
 
 
+#==============================================================================#
+#
+
+def get_theo_bandwidth_txns():
+
+    peak_bandwidths=[]
```

Comment:
Instead of `get_theo` and hard-coding, it should be able to be set from the command line options. Also, either implement this really long block from cs-roofline-toolkit or remove it.

Reply:
I have changed this so that it reads theoretical values from the command line. I have removed the commented block of code, I will work on getting these ceilings from the ERT and then add this.